lib/item_widget.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:lists/item.dart';
+
```

Comment:
I would recommend creating a directory called `widgets` or `view` to hold your widget files.